agents/python/opamp/health_status.py

```diff
@@ -4,4 +4,5 @@ class AgentHealthStatus(str, Enum):
     HEALTHY = "Healthy"
     STARTING = "Starting"
     UNSUPPORTED_RUNTIME_VERSION = "UnsupportedRuntimeVersion"
-    TERMINATED = "ProcessTerminated"
\ No newline at end of file
+    TERMINATED = "ProcessTerminated"
+    AGENT_FAILURE = "AgentFailure"
```

Comment:
Please also add it [here](https://github.com/odigos-io/odigos/blob/880f2ce4febd5309048dccb7caad737908fd0991/common/agent_health_status.go#L25) 🙏